github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/help.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Trigger that run job for GitHub Pull Request activities.
+    Trigger that launches builds in response to GitHub Pull Request activities.
```

Comment:
may be word integrates would sound good... Ideally jenkins integrates with GH PR functionality and allows trigger builds (better call runs).